pkg/locodedb/record.go

```diff
@@ -0,0 +1,50 @@
+package locodedb
+
+import (
+	"fmt"
+)
+
+// Key represents the key in location database. It contains the country code and the location code.
+type Key struct {
+	cc *CountryCode
+	lc *LocationCode
+}
+
+// NewKey returns a new Key from a country code and a location code string pair (e.g. "USNYC") or an error if the string is invalid. The country code must be 2 letters long and the location code 3 letters long.
+func NewKey(lc [2]string) (*Key, error) {
```

Comment:
But why not `NewKey(country, location string)(*Key, error)`? You have to do some dances with this API in nspcc-dev/neofs-node#2634 like
```
locodedb.NewKey([2]string{"RU", "MOW"})
...
newLocode := [2]string{lc[:2], lc[2:]}
locodedb.NewKey(newLocode)
```
Compare with
```
locodedb.NewKey("RU", "MOW")
...
locodedb.NewKey(lc[:2], lc[2:])
```